controls/3.6.rb

```diff
@@ -53,14 +53,21 @@
     kms_cache.kms_key_ring_names[location].each do |keyring|
       kms_cache.kms_crypto_keys[location][keyring].each do |keyname|
         key = google_kms_crypto_key(project: gcp_project_id, location: location, key_ring_name: keyring, name: keyname)
-        rotation_period_int = key.rotation_period.delete_suffix('s').to_i
-        next unless key.primary_state == "ENABLED"
-        describe "[#{gcp_project_id}] #{key.crypto_key_name}" do
-          subject { key }
-          it "should have a lower or equal rotation period than #{kms_rotation_period_seconds}" do
-            expect(rotation_period_int).to be <= kms_rotation_period_seconds
+        if key.rotation_period.nil?
+          describe "[#{gcp_project_id}] #{key.crypto_key_name}" do
+            subject { key }
+            its('rotation_period') { should_not be_nil }
+          end
+        else
+          rotation_period_int = key.rotation_period.delete_suffix('s').to_i
+          next unless key.primary_state == "ENABLED"
```

Comment:
we don't need to check Disabled keys at all, this probably belongs at the top of the loop

Reply:
Agreed and fixed. :shipit: